docs/.vitepress/theme/custom.css

```
@@ -162,7 +259,16 @@ kbd {
   border: 1px solid var(--vp-c-border);
   padding: 0.15rem 0.25rem;
   border-radius: 4px;
-  color: var(--vp-c-brand-3);
+  color: var(--vp-c-text-1);
+}
+
+.VPFeature :not(pre) > code,
+.VPHero :not(pre) > code {
+  background-color: rgba(255, 229, 100, 0.2);
```

Stylelint (17.2.0)

[error] Expected "0.2" to be "20%" (alpha-value-notation)

(alpha-value-notation)